neptune_tensorflow_keras/impl/__init__.py

```diff
@@ -130,7 +130,7 @@ def _log_metrics(self, logs, category: str, trigger: str):
                 pass
 
     def on_train_begin(self, logs=None):  # pylint:disable=unused-argument
-        self._metric_logger["training/model/summary"] = _model_summary_file(self.model)
```

Comment:
Should we update `self._metric_logger` to `self.base_handler` or something else? As we use it log model summary and metrics (and also model checkpoint in future).

Reply:
Why?

Reply:
I meant the name, sorry. (as it is used for more than metric logging like logging model summary). 

I see it as a handler to base namespace so maybe `self.base_namespace_handler`.

Reply:
Right now it's used only once, I'll stick with the rule of thumb that it should be refactored if it repeats at least three times.